compiler/damlc/daml-doc/src/DA/Daml/Doc/Extract/Templates.hs

```diff
@@ -46,8 +48,9 @@ getTemplateDocs DocCtx{..} typeMap templateInstanceMap =
       , td_payload = getFields tmplADT
       -- assumes exactly one record constructor (syntactic, template syntax)
       , td_choices = map mkChoiceDoc choices
-      -- is filled via distributeInstanceDocs
```

Comment:
since we no longer have `Implements` instances, it doesn't make sense to do this via `distributeInstanceDocs`. Instead, this function now takes a new argument to map template types to the interface types they implement.